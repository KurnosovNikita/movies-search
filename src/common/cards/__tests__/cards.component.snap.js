import React from 'react';
import {shallow} from 'enzyme';
import Cards from '../cards.component';

describe('Cards component', () => {
    let props;
    beforeEach(() => {
        props = {
            movies: [],
            handleImageClick: jest.fn(),
        };
    });

    it('should render <Cards> correctly', () => {
        const component = shallow((<Cards {...props} />));

        expect(component).toMatchSnapshot();
    });
});
