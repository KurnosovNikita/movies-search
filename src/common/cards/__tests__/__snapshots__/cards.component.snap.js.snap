// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards component should render <Cards> correctly 1`] = `
<div
  className="cards-container"
/>
`;
